lib/open_pantry/web/views/user_view.ex

```diff
@@ -24,6 +24,7 @@ defmodule OpenPantry.Web.UserView do
     |> Enum.map(&({&1.english_name, &1.id}))
   end
 
+  # Copied directly from OpenPantry.Web.UserSelectionView
```

Comment:
I think can just use defdelegate here, or possibly import the function from the UserSelectionView, assumign this is actually needed...  I'm a little fuzzy on why this is needed here now, or if still needed there, but just FYI